ProcessLib/NumericsConfig.h

```diff
@@ -65,12 +77,20 @@ namespace detail
 #endif    // USE_LIS
 #endif    // OGS_USE_EIGEN
 
-
 //
 // Global vector/matrix builder.
 //
 
-#include "AssemblerLib/SerialVectorMatrixBuilder.h"
+#ifdef USE_PETSC
```

Comment:
createVector and createMatrix in PETScVectorMatrixBuilder have different argument compared to that of the serial version.